legal-api/migrations/versions/33501a263f32_sp_gp.py

```diff
@@ -0,0 +1,58 @@
+"""sp_gp
+
+Revision ID: 33501a263f32
+Revises: 94586877d4bc
+Create Date: 2022-02-28 12:53:01.294444
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '33501a263f32'
+down_revision = '94586877d4bc'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('businesses', sa.Column('business_start_date', sa.DateTime(timezone=True), nullable=True))
+    op.add_column('businesses', sa.Column('naics_key', sa.String(length=50), nullable=True))
+    op.add_column('businesses', sa.Column('naics_code', sa.String(length=10), nullable=True))
+    op.add_column('businesses', sa.Column('naics_title', sa.String(length=150), nullable=True))
+    op.add_column('businesses_version', sa.Column('business_start_date', sa.DateTime(timezone=True), nullable=True))
+    op.add_column('businesses_version', sa.Column('naics_key', sa.String(length=50), nullable=True))
+    op.add_column('businesses_version', sa.Column('naics_code', sa.String(length=10), nullable=True))
+    op.add_column('businesses_version', sa.Column('naics_title', sa.String(length=150), nullable=True))
```

Comment:
maybe name this `naics_description` instead.  in the case where a valid naics code is provided, this will be populated with the naics title but the manual entry will also get populated here too right?  I think it might be useful to have a more generic name for this column.